src/platform/platform_worker.c

```diff
@@ -149,8 +149,12 @@ CxPlatWorkersLazyStart(
         goto Error;
     }
 
+    BOOLEAN SetIdealProc = !(Config->Flags & QUIC_EXECUTION_CONFIG_FLAG_NO_IDEAL_PROC);
```

Comment:
`Config` may be `NULL`.
```suggestion
    BOOLEAN SetIdealProc = !Config || !(Config->Flags & QUIC_EXECUTION_CONFIG_FLAG_NO_IDEAL_PROC);
```